suite-native/module-accounts/src/components/AccountDetailHeader.tsx

```diff
@@ -1,19 +1,53 @@
-import React, { memo } from 'react';
+import React, { memo, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 
+import { Graph, TimeSwitch } from '@suite-native/graph';
 import { AccountBalance } from '@suite-native/accounts';
 import { Box, Text } from '@suite-native/atoms';
 import { AccountKey } from '@suite-common/suite-types';
+import {
+    getSingleAccountGraphPointsThunk,
+    LineGraphTimeFrameValues,
+    selectAccountGraphPoints,
+} from '@suite-common/wallet-graph';
+import { selectFiatCurrency } from '@suite-native/module-settings';
 
 type AccountDetailHeaderProps = {
     accountKey: AccountKey;
     accountName?: string;
 };
 
-export const AccountDetailHeader = memo(({ accountKey, accountName }: AccountDetailHeaderProps) => (
-    <>
-        <AccountBalance accountKey={accountKey} accountName={accountName} />
-        <Box marginBottom="large">
-            <Text variant="titleSmall">Transactions</Text>
-        </Box>
-    </>
-));
+export const AccountDetailHeader = memo(({ accountKey, accountName }: AccountDetailHeaderProps) => {
+    const dispatch = useDispatch();
```

Comment:
nit: would be great to have all state-related hooks (`useSelect` + `useState` ) at the top of components, since they are usually the most important ones to component logic

Reply:
I agree that we shouldn't really pay too much attention to hooks ordering. If we would, we should definitely have this taken care of by eslint configuration and not do it manually. But I'm quite happy to have this somewhat flexible.

Reply:
Cases where you need hooks before state because their output are exceptions of course, it doesn't happen that often. Sure, if you agreed to put keep dispatch at the beginning so that you can easily ignore it it's ok but hierarchically state hooks are still the most important ones 😓  We could use this [plugin](https://github.com/hiukky/eslint-plugin-hooks) for the rules, I did not want to be too draconian with ordering rules since the whole thing is quite niche. On the other hand if no one pays attention to hook ordering code becomes harder to read because the patterns are sometimes too different.

Reply:
Yeah, if we agree on this, I would be happy to implement this rule myself, if you wouldn't mind @dahaca? I actually have this as one of my goals set for this year so I wouldn't mind setting this up. 

Only reason why I want to have this outsourced to ESLint is kind of developer laziness. I always order it just so it looks good to me when I read it but understand everyone has their own preferences. This would make it easier for me (and everyone else) so that we don't have to order it at all manually but eslint will do it, same as with import ordering.

Reply:
I am not fan of enforcing specific order. Sometimes it just doesn't make sense and if this autofixed it's hard to disable it.